editor/editor.tsx

```diff
@@ -21,9 +21,14 @@ import './editor.css';
 import { ICONS, ICON_INFO } from './icon';
 import type { HiGlassSpec } from '@higlass.schema';
 import type { Datum } from '@gosling.schema';
+import * as ts from 'typescript';
 // @ts-ignore
 import { Themes } from 'gosling-theme';
 
+function json2js(jsonCode: string) {
+    return `var spec=${jsonCode}`;
```

Comment:
Just to use the consistent formatting with your JS codes.
```suggestion
    return `var spec = ${jsonCode}`;
```